packages/terra-button/src/Button.module.scss

```diff
@@ -24,16 +24,19 @@
   // stylelint-enable selector-no-qualifying-type
 
   .button {
+    align-items: center;
     border-radius: var(--terra-button-border-radius, 0.2142rem);
     border-style: solid;
     border-width: 1px;
     cursor: pointer;
-    display: inline-block;
+    display: inline-flex;
     font-family: inherit;
     font-size: var(--terra-button-font-size, 1rem);
+    justify-content: center;
     line-height: var(--terra-button-line-height, 1.429);
     margin: 0; /* Reset margin for Safari. Safari defaults to 2px. */
     max-width: 100%; /* Need to help with overflow-wrap */
+    min-height: 2rem;
```

Comment:
I noticed some of the icon buttons went from being 30px tall to 28px tall. I think we should set this min-height to `2.1428rem` to help ensure the buttons stay 30px tall, the same height as non-icon buttons.

Reply:
Resolved in 2ff8a50